src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimelinePoint.js

```diff
@@ -5,27 +5,9 @@ import { selectors } from "../../../../../../../ui/reducers";
 const { getAnalysisPointsForLocation } = selectors;
 import { actions } from "ui/actions";
 import { Marker } from "ui/components/Timeline/Message";
-import { getThreadExecutionPoint } from "../../../reducers/pause";
+import { getExecutionPoint } from "../../../reducers/pause";
 import { timelineMarkerWidth as pointWidth } from "../../../../../../../ui/constants";
 
-function getClassnameObject(point, executionPoint) {
-  const classObj = { past: false, pause: false, future: false };
-
-  if (!executionPoint) {
-    return classObj;
-  }
-
-  if (BigInt(point) < BigInt(executionPoint)) {
-    classObj.past = true;
-  } else if (BigInt(point) > BigInt(executionPoint)) {
-    classObj.future = true;
-  } else {
-    classObj.pause = true;
```

Comment:
this is an elaborate example of using booleans when you can just use the value of the expression